presto/presto_test.go

```diff
@@ -208,6 +240,33 @@ func TestQueryFailure(t *testing.T) {
 	}
 }
 
+func TestSSLCertPath(t *testing.T) {
+	db, err := sql.Open("presto", "https://localhost:9?SSLCertPath=/tmp/invalid_test.cert")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer db.Close()
+
+	want := "Error loading SSL Cert File"
+	if err := db.Ping(); err == nil {
+		t.Fatal(err)
+	} else if !strings.Contains(err.Error(), want) {
+		t.Fatalf("want: %q, got: %w", want, err)
```

Comment:
This requires Go 1.13 or newer because of %w, right?

If you really want to use %w then we'd have to update the .travis.yaml file to use Go 1.13 and newer.

Reply:
It could be easier to just replace %w with %v instead.

Reply:
Thanks for the review @fiorix
I think for now we can replace with %v